src/services/ui/src/pages/welcome/index.tsx

```diff
@@ -82,37 +107,50 @@ export const Welcome = () => {
                 />
                 <C.Step
                   icon={<AcademicCapIcon className="min-w-[32px] w-8 h-8" />}
-                  title="Possimus a, odio."
-                  content="Lorem ipsum dolor sit amet."
+                  title="Login with EUA"
+                  content="Login with your EUA username and password to access the SPA and Waiver dashboard."
                 />
                 <C.Step
                   icon={<AcademicCapIcon className="min-w-[32px] w-8 h-8" />}
-                  title="Possimus a, odio."
-                  content="Lorem ipsum dolor sit amet."
+                  title="Review your assigned submission"
+                  content="Search the submission ID from the email and click on the submission to view and review details and attachments."
                 />
               </C.HowItWorks>
               <div>
-                <h4 className="font-bold text-xl mb-4">Lorem, ipsum.</h4>
+                <h4 className="font-bold text-xl mb-4">
+                  Submission Types include:
+                </h4>
                 <ul className="flex flex-col gap-4">
+                  <li>
+                    <p>Amendments to your Medicaid and CHIP State Plans.</p>
+                  </li>
+                  <li>
+                    <p>
+                      Official state responses to formal requests for additional
+                      information (RAIs) for SPAs.
+                    </p>
+                  </li>
+                  <li>
+                    <p>Section 1915(b) waiver submissions.</p>
+                  </li>
+                  <li>
+                    <p>Section 1915(c) Appendix K amendments.</p>
+                  </li>
                   <li>
                     <p>
-                      Lorem ipsum dolor sit amet consectetur adipisicing elit.
-                      Architecto facilis laboriosam placeat molestias animi
-                      atque sint modi accusantium maiores. Aliquam sequi ad
-                      nobis nesciunt dignissimos natus dolorum quo illum vel?
+                      Official state responses to formal requests for additional
+                      information (RAIs) for Section 1915(b) waiver actions.
                     </p>
                   </li>
                   <li>
                     <p>
-                      Lorem ipsum dolor sit amet consectetur adipisicing elit.
-                      Architecto facilis laboriosam placeat molestias animi
-                      atque sint modi accusantium maiores. Aliquam sequi ad
-                      nobis nesciunt dignissimos natus dolorum quo illum vel?
+                      State requests for Temporary Extensions for section
+                      1915(b) and 1915(c) waivers.
                     </p>
                   </li>
                 </ul>
```

Comment:
at the end of the pade can we add a "Do you have questions or need support? [button]" that links to the faq page (in another tab). it doesnt need to have the fancy background gradient thing like onemac